owasp-top10/profile_service/app.py

```
@@ -0,0 +1,68 @@
+from flask import Flask, request, jsonify
+import requests
+from pymongo import MongoClient
+import pickle
```

Ruff (0.8.0)

`pickle` imported but unused

Remove unused import: `pickle`

(F401)